docgen/start.js

```diff
@@ -17,6 +17,7 @@ builder({middlewares}, err => {
 
 // then we watch and rebuild
 watch([
+  rootPath('packages/react-instantsearch/src/**/*.js'),
```

Comment:
Now there is content in the js files, that's why we should trigger the build of the jsdoc (which is in the metalsmith build)